src/app/components/drawer/Projects/ArticlesComponent.js

```diff
@@ -1,12 +1,145 @@
 import React from 'react';
+import { Link } from 'react-router';
+import { FormattedMessage } from 'react-intl';
+import cx from 'classnames/bind';
+import PublishedIcon from '../../../icons/fact_check.svg';
+import FileDownloadIcon from '../../../icons/file_download.svg';
+import BookIcon from '../../../icons/book.svg';
 import styles from './Projects.module.css';
+import { publishedDefaultQuery } from '../../team/Published';
+import ProjectsCoreListCounter from './ProjectsCoreListCounter';
+import NewArticleButton from './NewArticleButton';
+import { importedReportsDefaultQuery } from '../../team/ImportedReports';
 
-const ArticlesComponent = () => (
-  <React.Fragment>
-    <div className={styles.listTitle}>
-      Articles
-    </div>
-  </React.Fragment>
-);
+const ArticlesComponent = ({ team }) => {
+  // Get/set which list item should be highlighted
+  const pathParts = window.location.pathname.split('/');
+  const [activeItem, setActiveItem] = React.useState({ type: pathParts[2], id: parseInt(pathParts[3], 10) });
+
+  React.useEffect(() => {
+    const path = window.location.pathname.split('/');
+    if (activeItem.type !== path[2] || activeItem.id !== path[3]) {
+      setActiveItem({ type: path[3], id: parseInt(path[3], 10) });
+    }
+  }, [window.location.pathname]);
+
+  const handleSpecialLists = (listId) => {
+    setActiveItem({ type: listId, id: null });
+  };
+
+  const menuOptions = [
+    { label: 'Claim & Fact Check', icon: <PublishedIcon /> },
+    { label: 'Explainer', icon: <BookIcon /> },
+  ];
+
+  const handleMenuItemClick = () => {};
```

Comment:
I think these don't need to be props, they could be moved to the `NewArticleButton` component